examples/shader/Effect.hpp

```diff
@@ -71,16 +68,14 @@ class Effect : public sf::Drawable
     }
 
 private:
-
     // Virtual functions to be implemented in derived effects
-    virtual bool onLoad() = 0;
-    virtual void onUpdate(float time, float x, float y) = 0;
+    virtual bool onLoad()                                                               = 0;
+    virtual void onUpdate(float time, float x, float y)                                 = 0;
     virtual void onDraw(sf::RenderTarget& target, const sf::RenderStates& states) const = 0;
```

Comment:
This seems like a rather unnecessary and easily overlooked alignment, but I can also live with it